### PR TITLE
Update mpas-source: CVMix and GM updates

### PR DESCRIPTION
**Pull request by mark-petersen**
Includes two MPAS PRs:
- Fixes and Updates to CVMix https://github.com/MPAS-Dev/MPAS-Model/pull/703
- adds new GM option -- Eden Greatbatch https://github.com/MPAS-Dev/MPAS-Model/pull/705

see links for more details.

[NML]
[non-bfb]

### CODE REVIEW

**Comment by jonbob** [comment]
@mark-petersen - did you revert the seaice commit from late last week?

**Comment by mark-petersen** [comment]
@jonbob this PR is ready for a namelist update. The mpas-source submodule is pointing to `e3sm/testing` branch, as @vanroekel wanted a little more time for testing. But the namelist flags will not change from what is here now.

I reverted the mpas changes for prognostic sea ice, both here and on `e3sm/develop`.

**Comment by jonbob** [comment]
OK, I'll run the scripts now

**Comment by jonbob** [comment]
OK, the scripts are updated and I've submitted a small sanity test (ERS.ne4_oQU240.A_WCYCL1850.anvil_intel), though it's stuck in the queue right now.

**Comment by jonbob** [comment]
@mark-petersen - is this good to go? I see that you're making commits to the MPAS code? If so, @vanroekel , can you please sign off on it? We're getting near the end fo the day...

**Comment by mark-petersen** [comment]
MPAS PRs are merged. @vanroekel please approve here as well, based on your testing.

File: components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml [hunk 1]
```diff
@@ -157,10 +157,12 @@
 <config_GM_kappa ocn_forcing="datm_forced_restoring" ocn_grid="EC30to60E2r2">600.0</config_GM_kappa>
 <config_GM_closure>'visbeck'</config_GM_closure>
 <config_GM_baroclinic_mode>3.0</config_GM_baroclinic_mode>
-<config_GM_visbeck_alpha>0.13</config_GM_visbeck_alpha>
-<config_GM_visbeck_max_depth>1000.0</config_GM_visbeck_max_depth>
-<config_GM_visbeck_min_kappa>300.0</config_GM_visbeck_min_kappa>
-<config_GM_visbeck_max_kappa>1200.0</config_GM_visbeck_max_kappa>
+<config_GM_Visbeck_alpha>0.005</config_GM_Visbeck_alpha>
```
Comment (vanroekel) [inline components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml:160]: can this be changed back to 0.13?  This is the value we've been using in coupled runs

**Review by vanroekel — CHANGES_REQUESTED** [review]
all looks good, but one default setting for visbeck got changed that should be reverted to its present value

File: components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml (hunk 1 above)
Comment (jonbob) [reply components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml:160]: @vanroekel - yes, I'll take care fo that right now

Comment (jonbob) [reply components/mpas-ocean/bld/namelist_files/namelist_defaults_mpaso.xml]: OK, should be good now

**Comment by mark-petersen** [comment]
@vanroekel any other comments, or can you approve?

**Review by vanroekel — APPROVED** [review]
looks good, thanks for the change!

**Comment by mark-petersen** [comment]
@jonbob this one is ready when you are.

**Review by jonbob — APPROVED** [review]
Looks good from a visual inspection

**Comment by jonbob** [comment]
merged to next -- with lots of help from @rljacob, @oksanaguba and @bishtgautam

**Comment by jonbob** [comment]
merged to master and expected DIFFs blessed

**Pull request merged**